modeldict/base.py

```diff
@@ -20,6 +25,9 @@ def __init__(self, cache=cache, timeout=30):
         self.remote_cache_key = cls_name
         self.remote_cache_last_updated_key = '%s.last_updated' % (cls_name,)
 
+        if not remote_timeout == DefaultRemoteTimeout:
```

Comment:
I would probably use `is` here since we really want to be doing an identity check.

Reply:
Fixed